hw2/css/style.css

```diff
@@ -0,0 +1,169 @@
+.clearfix:after {
+	content: '';
+	clear: both;
+	display: table;
+	overflow: hidden;
+}
+
+a{
+	text-decoration: none;
+}
+
+img{
+	width: 100%;
+}
+
+body{
+	background-color: #f9f9f9;
+}
+
+	.container{
+		background-color: #FFF;
+		margin: 0 auto;
+		width: 78%;
+		box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+		padding: 50px 48px 60px 46px;
+	}
+
+		.header__logo{
+			width: 145px;
+			height: 40px;
+			margin-bottom: 40px;
+		}
+
+		.menu-top{
+			display: flex;
+			line-height: 40px;
+			width: 100%;
+			background-color: #fff;
+		    background-image: -webkit-linear-gradient(bottom, #444343 0%, #5e5e5e 100%);
+		    background-image: -o-linear-gradient(bottom, #444343 0%, #5e5e5e 100%);
+		    background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%);
+		}
+
+			.menu-top__item{
+				text-shadow: 5px 5px 7px rgba(0, 0, 0, 0.8);
+				font-family: "Open Sans", "Fira Sans", Arial, sans-serif;
+				padding: 0 4%;
+				color: #fff;
+				box-sizing: border-box;
+    			border: 1px solid rgba(0, 0, 0, 0);
+			}
+
+				.menu-top__item:hover{
+					background-color: #fff;
+				    background-image: -webkit-linear-gradient(bottom, #2e2e2e 0%, #4a4a4a 100%);
+				    background-image: -o-linear-gradient(bottom, #2e2e2e 0%, #4a4a4a 100%);
+				    background-image: linear-gradient(to top, #2e2e2e 0%, #4a4a4a 100%);
+				    border-right: 1px solid #838383;
+					border-left: 1px solid #838383;
+					box-sizing: border-box;
+				}
+
+		.main-content{
+			margin-top: 40px;
+			display: inline-block;
+			width: 28%;
+		}
+
+			.main-content__menu{
+				display: inline-block;
```

Comment:
Otherwise, his menu items has width only by their content, unlike from markup. needed for exact displaying as at the maquette.